src/popup/basics/Forms.tsx

```diff
@@ -0,0 +1,155 @@
+
+import { COLOR_PALETTE } from "popup/constants/styles";
+import { Button } from "popup/basics/Buttons";
+
+interface FormProps {
+  children: React.ReactNode;
+  className?: string;
+}
+
+const StyledFormEl = styled(FormikForm)`
+  display: flex;
+  flex-flow: column wrap;
+`;
+
+export const Form = ({ children, className }: FormProps) => (
+  <StyledFormEl className={className}>{children}</StyledFormEl>
+);
+
+const FormButtonEl = styled(Button)`
+  &:disabled {
+    color: ${COLOR_PALETTE.secondaryText};
+  }
+`;
+
+interface SubmitButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  children: React.ReactNode;
+  isSubmitting?: boolean;
+  isValid?: boolean;
+  size?: string;
+  onClick?: () => void;
+}
+
+export const SubmitButton = ({
+  children,
+  isSubmitting = false,
+  isValid = true,
+  onClick = () => {},
+  size,
+  ...props
+}: SubmitButtonProps) => (
+  <FormButtonEl
+    {...props}
+    onClick={onClick}
+    size={size}
+    type="submit"
+    disabled={isSubmitting || !isValid}
+  >
+    {isSubmitting ? "Loading..." : children}
+  </FormButtonEl>
+);
+
+/* Form */
+interface ErrorMessageProps {
+  error: React.ReactNode;
+}
+
+const FormErrorEl = styled.div`
+  color: ${COLOR_PALETTE.error};
+  font-size: 0.8125rem;
+  height: 1rem;
+  padding: 0.25rem 0 0.75rem;
+  text-align: center;
+  line-height: 1;
+`;
+
+const FormCheckBoxWrapperEl = styled.div`
+  display: inline-block;
+  margin-right: 0.625rem;
+`;
+
+export const ApiErrorMessage = ({ error }: ErrorMessageProps) =>
+  error ? <FormErrorEl>{error}</FormErrorEl> : null;
+
+export const FormRow = styled.div`
+  position: relative;
+  padding: 0.2rem 0;
+  max-width: 24.5rem;
+  width: 100%;
+`;
+
+export const Error = ({ name }: { name: string }) => (
+  <FormErrorEl>
+    <ErrorMessage name={name} component="span" />
+  </FormErrorEl>
+);
+
+export const TextField = styled(Field)`
+  border-radius: 1.25rem;
+  border: ${(props) =>
+    props.hasError ? `1px solid ${COLOR_PALETTE.error}` : 0};
+  box-sizing: border-box;
+  background: ${COLOR_PALETTE.inputBackground};
+  font-size: 1rem;
+  padding: 1.875rem 2.25rem;
+  width: 100%;
+  resize: none;
+
+  &::-webkit-input-placeholder {
+    font-family: "Muli", sans-serif;
+  }
+
+  &:-ms-input-placeholder {
+    font-family: "Muli", sans-serif;
+  }
+
+  &:-moz-placeholder {
+    font-family: "Muli", sans-serif;
+  }
+
+  &::-moz-placeholder {
+    font-family: "Muli", sans-serif;
+  }
+`;
+
+const FormCheckboxFieldLabelEl = styled.label`
+  div {
+    border-radius: 2rem;
+    height: 1rem;
+    width: 1rem;
+  }
+`;
+
+const FormCheckboxFieldEl = styled(Field)`
+  align-items: center;
+  background: ${COLOR_PALETTE.inputBackground};
+  border: 0.125rem solid ${COLOR_PALETTE.inputBackground};
+  border-radius: 0.625rem;
+  color: ${COLOR_PALETTE.primary};
+  cursor: pointer;
+  display: flex;
+  height: 2rem;
+  justify-content: space-evenly;
+  width: 2rem;
+
```

Comment:
Looks like you're reimplementing checkboxes, but this doesn't match the design anymore. Let's leave this PR as just styleguide changes and reimplement checkboxes separately
<img width="418" alt="Screen Shot 2020-07-17 at 9 09 16 PM" src="https://user-images.githubusercontent.com/6789586/87841245-04c3f700-c872-11ea-8210-d349ff67b5f4.png">

Reply:
Oops! That was an exploration but I meant to back it out